Document module deprecation in 2.x (#2855)
apache/logging-log4j2: src/site/antora/modules/ROOT/pages/manual/appenders/message-queue.adoc

[[JeroMqAppender]]
== ZeroMQ/JeroMQ Appender

[WARNING]
====

> *This appender is planned to be removed in the next major release, due to lack of maintainers!*
> 
> Users might consider switching to the
> https://github.com/fbacchella/loghublog4j2#zmqappender[third-party `ZMQ` appender].

Reviewer: ```suggestion
*This appender is planned to be removed in the next major release!*
Users should consider switching to
https://github.com/fbacchella/loghublog4j2#zmqappender[a third-party `ZMQ` appender].
```
